dart-templates/serialization/native/class.mustache

```diff
@@ -281,7 +281,13 @@ class {{{classname}}} {{#interfaces}}{{#-first}}implements {{/-first}}{{.}}{{^-l
             {{name}},
           {{/isContainer}}
           {{^isContainer}}
-            {{name}}{{^required}}?{{/required}}.toJson(),
+            {{name}}{{^required}}?{{/required}}.toJson(keys?.fold<List<String>>(<String>[], (List<String> previousValue, String element) {
+              if (element.startsWith(r'{{{name}}}.')) {
+                previousValue.add(element.split(r'{{{name}}}.').last);
```

Comment:
Why not something like : 
`previousValue.add(element.substring(r'{{{name}}}.'.length));`

(Stupid) examples which might cause trouble : 
- keys = ['child.child.name']
- keys = ['adress.parent.adress.city']
- keys = ['tree.subtree.name']

Once again I know it's unlikely it will happen but who knows :) ?

Reply:
your comment is on a outdated file, the current code uses `RegExp(r'^{{{name}}}\.')` so none of these examples will cause trouble.

Reply:
Just saw that, but my solution seems simpler :)